### ISSUE TITLE
spurious files (dev/, some prov/serializers/) in released pypi tarballs

### ISSUE THREAD

**Issue by yarikoptic** [trungdong/prov#106]
While reviewing a Debian package for 1.5.0 release I have noted that there are some files in the tarball (fetched from pypi) which aren't within this git repository for 1.5.0 release/tag. 

```shell
diff -Naur --exclude=.git prov 1.5.0-1/python-prov-1.5.0 | diffstat | grep -v -e 'debian/' -e '\.pc/' -e 'egg-info' -e PKG-I | grep '\+'
 dev/debug_jsons.py                   |   58 ++
 dev/script.py                        |   99 +++
 dev/test_count_JSON-LD.py            |   38 +
 dev/test_count_Streaming.py          |   82 +++
 prov/serializers/jsonld_streaming.py |  209 +++++++
 prov/serializers/provjsons.py        |  368 ++++++++++++++
```

my guess is that release tarball was generated on a development box in a non-clean git repository so those files got picked up as well... just wanted to let you know.

(since git clones are so cheap, paranoid me usually quickly clones the repo for running 'sdist' command ;) )

**Comment by trungdong** [comment]
Good catch! Thank you. and sorry for the delay in responding.
I'm working on a fix release (1.5.1) and will remove those in that.

**Comment by trungdong** [comment]
@yarikoptic 
I took the advice and cloned before distributed the new release. Hope this is now fixed.
Thanks again!
